.github/workflows/linux-32bit-build-and-test.yaml

```diff
@@ -8,11 +8,13 @@ name: Regression Linux i386
       - '**.md'
       - 'LICENSE*'
       - NOTICE
+      - '.github/workflows/*.yaml'
```

Comment:
This will prevent us from testing workflow changes in a PR, no?

Reply:
You're correct... if we need for some reason to change the regression workflow itself it will be ignored... will remove this part and leave just with the name of the ignored regression that was wrong.